client/src/Components/Reviews.jsx

```diff
@@ -8,16 +8,53 @@ class Reviews extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      load: false,
       mostPopular: [],
       allReviews: [],
     };
+    this.vote = this.vote.bind(this);
+  }
+
+  vote(e) {
+    e.persist();
```

Comment:
What does this do, and why is it necessary? (Genuine question--I'm not saying that it's not necessary.)